examples/fetch/index.tsx

```diff
@@ -23,11 +23,55 @@ const App = () => {
     }
   };
 
+  async function streamToArrayBuffer(stream: any) {
+    const reader = stream.getReader();
+    while (true) {
+      console.log("await read");
+      const { done, value } = await reader.read();
+      console.log("done", done);
+      if (done) {
+        break;
+      } else {
+        console.log(value.byteLength);
+        // @ts-ignore
+        const text = globalThis.TextCodecHelper.decode(value);
```

Comment:
It doesn't seem to be a public API.

https://github.com/llong2195/lynx/blob/945636d758be85c3ddf802ee054d68f3a57034c1/core/runtime/bindings/jsi/global.cc#L47

Can we instead provde a `TextDecoder` instead?

Or we may use a polyfill instead of relying on an internal API of Lynx in our example.